Description: Disables the test for the upstream issue BCEL-336 which involves a malformed class file not available in Debian
Author: Emmanuel Bourg <ebourg@apache.org>
Forwarded: not-needed
--- a/src/test/java/org/apache/bcel/generic/MethodGenTestCase.java
+++ b/src/test/java/org/apache/bcel/generic/MethodGenTestCase.java
@@ -26,8 +26,6 @@
 import org.junit.Assert;
 import org.junit.Test;
 
-import javax.mail.internet.MailDateFormat;
-
 public class MethodGenTestCase {
 
     @interface A {
@@ -121,6 +119,7 @@
     }
 
     @Test(expected = IllegalStateException.class)
+    @org.junit.Ignore
     public void testInvalidNullMethodBody_MailDateFormat() throws Exception {
         testInvalidNullMethodBody("javax.mail.internet.MailDateFormat");
     }
